collectors/ebpf.plugin/README.md

```diff
@@ -39,18 +39,93 @@ section for details.
 
 ## Charts
 
-The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with three sub-menus: **File**, **VFS**, and
-**Process**. All the charts in this section update every second. The collector stores the actual value inside of its
-process, but charts only show the difference between the values collected in the previous and current seconds.
+The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with two sub-menus: **Socket**, and
+**Process**. The plugin also creates charts on different menus, like System Overview, Memory, Disks, Filesystem, 
+Mount Points and Applications. All the charts in this section update every second.
+The collector stores the actual value inside of its process, but charts only show the difference between the values
+collected in the previous and current seconds.
 
-### File
+### System Overview
 
-This group has two charts demonstrating how software interacts with the Linux kernel to open and close file descriptors.
+All charts on system overview are not enabled by default, because they would consume a lot of resources.
+See the [configuration](#configuration) section for details about how to enable them.
+
+#### Swap
+
+Inside swap submenu eBPF plugin creates the chart `swapcalls`, this chart is demonstrating when processes are calling
+functions `swap_readpage` and `swap_writepage`, functions responsible for doing IO in swap memory.
+
+### Memory
+
+In memory submenu eBPF plugin creates two submenus **page cache** and **synchronization**.
+
+#### Page cache
+
+This first submenu brings four charts that demonstrate statistics about page cache access.
+
+Page cache is adding `kprobes` for the kernel functions `add_to_page_cache_lru`, `mark_page_accessed`,
+`account_page_dirtied`, and `mark_buffer_dirty`. Netdata monitors page cache access giving an overall overview and also
+per application access.
+
+##### Page cache ratio
+
+The chart `cachestat_ratio` shows how processes are accessing page cache. In a normal scenario, we expect values around 
+ 100%, which means that the majority of the work on the machine is processed in memory.
+
+##### Dirty pages
+
+On `cachestat_dirties` netdata demonstrates the number of pages that were modified. This chart reflects calls to function
+`mark_buffer_dirty`.
+
+##### Page cache hits
+
+Netdata calls hits when an accessed page cache was not modified and we do not count pages that were added recently. This
+chart uses data from the four functions we are monitoring, mentioned above,.
+
+##### Page cache misses
+
+A page cache miss means that a page was not inside memory when the process tried to access it. This chart shows the result
+of the difference for calls between functions `add_to_page_cache_lru` and `account_page_dirtied`.
+
+#### Syncrhonization
+
+The **synchronization** submenu Netdata monitors calls
```

Comment:
Either `The **synchronization** submenu OF Netdata monitors calls...` or `IN the **synchronization** submenu, Netdata monitors...`

Reply:
also `calls for different syscalls` sounds confusing to me. Probably would put it as `monitors usage of various kernel syscalls`